containers/datalab/debug.sh

```diff
@@ -18,33 +18,24 @@
 # command prompt, which is useful for tinkering within the container before
 # manually starting the server.
 
-# In local mode the container picks up local notebooks, so it can be used
-# to work on files saved on the file system.
-
+CONTENT=$HOME
 ENTRYPOINT="/datalab/run-debug.sh"
-if [ "$1" == "shell" ]; then
-  ENTRYPOINT="/bin/bash"
+if [ "$1" != "" ]; then
```

Comment:
No, we may still want to do this. E.g. if we run the debugger inside the container.